services/export/shared/csvExport.js

```diff
@@ -300,8 +301,12 @@ const downloadTransactionHistory = async ({ filename }) => {
 		meta: {},
 	};
 
-	const isFileExists = await staticFiles.exists(filename);
-	if (!isFileExists) throw new NotFoundException(`File ${filename} not found.`);
+	if (!CSV_EXPORT_FILENAME.test(filename)) {
```

Comment:
Just so that it is clear and intuitive when reading through the code 🙂 
```suggestion
	if (!regex.CSV_EXPORT_FILENAME.test(filename)) {
```